src/FormattedMessage.js

```diff
@@ -1,5 +1,6 @@
 import React from "react";
 import PropTypes from "prop-types";
+import { TextContext } from "./TextProvider";
```

Comment:
No need of space before and after `TextContext`. Should be `import {TextContext} from "./TextProvider";`